swan-cern/files/swan_computing_config.py

```diff
@@ -275,7 +277,7 @@ def _gpu_enabled(self):
         """
         return True if the user has requested a GPU
         """
-        return "cu" in self.spawner.user_options[self.spawner.lcg_rel_field]
+        return "cu" in self.spawner.user_options.get(self.spawner.lcg_rel_field, '')
```

Comment:
This is correct, but I'm thinking of the best way to do this. Didn't we say at some point that GPUs would become a first class citizen in the web form, meaning they would appear separately from memory and cpu, and therefore here we could just check if the form had a GPU selected? Instead of relying on the name of the LCG release to contain "cu".

Reply:
yeah we meantioned that for the future, but that wouldn't be for now, right?

Reply:
Cause that would involve changes in all repos, including charts, gitops, spawner, etc.
And we wouldnt want to conflict all those changes with the customenvs for now

Reply:
The future is here: https://its.cern.ch/jira/browse/SWAN-215.

So yes, we leave it as it is here for now.